setup.py

```diff
@@ -42,8 +42,7 @@
     import pypandoc
     long_description = pypandoc.convert(readme_md, 'rst')
 except (IOError, ImportError):
-    print("Warning: pypandoc module not found, or pandoc not installed. " +
-          "Using md instead of rst")
+    print("Warning: pypandoc module not found, or pandoc not installed. Using md instead of rst")
```

Comment:
I would leave this code as it is, which is the same of what is in [perceval](https://github.com/chaoss/grimoirelab-perceval/blob/master/setup.py#L46).

Reply:
Actually it is giving flake8 error, so thats why modified it..

Reply:
`./setup.py:45:75: W504 line break after binary operator`

Reply:
Ok, I'll remove the binary operator, let me know for any other changes.

Reply:
yes, perfect! please align the code to https://github.com/chaoss/grimoirelab-perceval/blob/master/setup.py#L46 , and put it in a different commit

Reply:
The other option is to leave as it is, but please add some explanation in the commit message. The point is that in the current status, there is no info about why this file has been modified. Thus, if in the future we check the git log, we will not know why this was done.

Reply:
Yes agreed, to keep a record, I have made it into a seperate commit @valeriocos

Reply:
Yes, agreed, to keep track, made a seperate commit @valeriocos.